codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2FrameReader.java

```diff
@@ -391,11 +396,6 @@ private void verifyContinuationFrame() throws Http2Exception {
             throw connectionError(PROTOCOL_ERROR, "Continuation stream ID does not match pending headers. "
                     + "Expected %d, but received %d.", headersContinuation.getStreamId(), streamId);
         }
-
-        if (payloadLength < flags.getPaddingPresenceFieldLength()) {
-            throw streamError(streamId, FRAME_SIZE_ERROR,
```

Comment:
Why this is removed instead of changing to `connectionError`?

Reply:
This flag is not used for CONTINUATION frames and as described in https://datatracker.ietf.org/doc/html/rfc9113#section-4.1 "Unused flags MUST be ignored on receipt ...".